pkgs/top-level/python-packages.nix

```diff
@@ -31751,6 +31751,8 @@ EOF
 
   distro = callPackage ../development/python-modules/distro { };
 
+  PyLD = callPackage ../development/python-modules/PyLD { };
```

Comment:
the packages somewhat follow alphabetic order but as you've noticed its a bit messy. Try to find a bit better place than just at the end.